### ISSUE TITLE
Múltiple cursors vs zones

### ISSUE THREAD

**Issue by eko234** [jmorag/kakoune.el#9]
I really love kakoune and I tried your package a year ago or so, and today it came to My mind, that maybe the package zones.el could conceptually and technically be a better choice than multiple cursors, what do You think?

**Comment by jmorag** [comment]
I agree that it's conceptually more suited for kakoune emulation to have zones vs cursors. However, I don't really have the motivation to rewrite this package now, since I use it every day in its current state and it works fine for me. Of course, I'm happy to look at a PR or answer any questions you have if you'd like to take a crack at it.

**Comment by Zoybean** [comment]
I'm mid-way through writing my own version of multiple-cursors (and my own kakoune/helix inspired keybindings), using zones. If I get it in a usable state, I'll link it here.
